modules/govuk_apt/manifests/package_blacklist.pp

```diff
@@ -0,0 +1,22 @@
+# == Class: Govuk_apt::Package_blacklist
+#
+# Blacklists specific packages from being able to be installed on the system.
+# This means, if a user tries to install a package using apt, the package
+# specified will appear as unavailable for installation.
+#
+# === Parameters:
+#
+#  [*packages*]
+#    An array of packages that you wish to blacklist.
+#
+class govuk_apt::package_blacklist (
+  $packages = [],
+){
+  validate_array($packages)
+
+  apt::pin { "blacklist_${packages}":
```

Comment:
If we add a second package to the govuk_apt::package_blacklist::packages property and run Puppet again, this class generates a second file in addition to the first one, for instance '/etc/apt/preferences.d/blacklist_package1.pref' containing 'Package: package1', and a second  '/etc/apt/preferences.d/blacklist_package1package2.pref' containing 'Package: package1 package2', I don't know if this is what is expected